src/tests/integration-test

```diff
@@ -246,7 +251,8 @@ class UDisksTestCase(unittest.TestCase):
         If cd is True, return the CD device, otherwise the hard disk device.
         """
         block = klass.udisks_block(partition, cd)
-        return klass.client.get_object(block.get_object_path()).get_filesystem()
+        return klass.client.get_object(
+            block.get_object_path()).get_filesystem()
```

Comment:
Yes I know the tools can be configured a plethora of different ways, but no such configuration exists for udisks.  I'll redo this PR to just fix the bug(s) that were fixed in the clean-up and not change the WS formatting pep 8 guidelines.